chain-evm/src/machine.rs

```diff
@@ -14,12 +14,27 @@ use std::rc::Rc;
 use evm::{
     backend::{Apply, ApplyBackend, Backend, Basic, Log, MemoryVicinity},
     executor::{MemoryStackState, StackExecutor, StackSubstateMetadata},
-    Config, Context, Runtime,
+    Config as EvmConfig, Context, Runtime,
 };
 use primitive_types::{H160, H256, U256};
 
 use crate::state::AccountTrie;
 
+/// EVM Configuration.
+pub type Config = EvmConfig;
```

Comment:
No need for naming gymnastics:
```suggestion
pub type Config = evm::Config;
```